packages/manager-components/src/components/atoms/title/title.component.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  ODS_THEME_COLOR_HUE,
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+
+import React from 'react';
```

Comment:
React import should be the first import to follow ESLint rules

Reply:
👍